libs/nucs/src/validator.rs

```diff
@@ -20,13 +27,25 @@ pub struct ValidationParameters {
     /// The maximum allowed chain length.
     pub max_chain_length: usize,
 
+    /// The maximum width of a policy.
+    pub max_policy_width: usize,
+
+    /// The maximum depth of a policy.
+    pub max_policy_depth: usize,
```

Comment:
What is the width vs the depth of a policy?

Reply:
My understanding is: 
* too deep: `op(op(op(...))))`
* too wide is `op() & op() & ... & op()`
Is this correct?